bitnami/mongodb-sharded/README.md

```diff
@@ -58,8 +58,8 @@ The following table lists the configurable parameters of the MongoDB chart and t
 | `global.imagePullSecrets`                     | Global Docker registry secret names as an array                                                                                                           | `[]` (does not add image pull secrets to deployed pods)  |
 | `global.storageClass`                         | Global storage class for dynamic provisioning                                                                                                             | `nil`                                                    |
 | `image.registry`                              | MongoDB image registry                                                                                                                                    | `docker.io`                                              |
-| `image.repository`                            | MongoDB Image name                                                                                                                                        | `bitnami/mongodb`                                        |
-| `image.tag`                                   | MongoDB Image tag                                                                                                                                         | `{TAG_NAME}`                                             |
+| `image.repository`                            | MongoDB Image name                                                                                                                                        | `bitnami/mongodb-sharded`                                |
+| `image.tag`                                   | MongoDB Image tag                                                                                                                                         | `4.2.8-debian-10-r24`                                    |
```

Comment:
Could you revert this line? We use `{TAG_NAME}` because our automated system updates the values.yaml with new images but won't change this line in the README. That would cause to be come out of sync very soon.

Reply:
Ah, I see. Thanks for the review.